CONTRIBUTING.md

```diff
@@ -0,0 +1,170 @@
+Thanks so much for helping with the development of Reservations! This guide will cover the process for helping out with the project and should allow you to get started quickly and easily. If you need any help with contributing, please e-mail `stc-reservations{at}yale{dot}edu` to contact the core team.
+
+## Getting Started
+There are a number of ways you can contribute to the Reservations project, from [reporting bugs](#bug-reports) and [making suggestions](#feature-requests), to [updating documentation](#improving-documentation) and [improving the codebase](#contributing-code). If you're new to coding in general, the first three options are a good way to familiarize yourself with the application and the overall codebase. If you're new to Rails development, feel free to work on some of our less complex issues, labeled with [`diff: 1`](https://github.com/YaleSTC/reservations/labels/diff%3A%201) or [`diff: 2`](https://github.com/YaleSTC/reservations/labels/diff%3A%202). If you have any questions please feel free to comment in the GitHub issues or contact the core team; we appreciate any and all help we can get!
+
+## Reporting an Issue
+If you've been using Reservations and [notice a bug](#bug-reports), or want to [suggest a new feature](#feature-requests), please open a [GitHub issue](https://github.com/YaleSTC/reservations/issues/new). Before doing so, however, please **search open / closed issues** to see if someone else has already opened a similar issue.
+
+### Bug Reports
+A bug is a *verifiable problem* that is caused by the Reservations code. Good bug reports allow us to quickly identify the root causes of problems and resolve them. Here are some guidelines for submitting bug reports:
+
+1. **Is it a duplicate?** - check to see if the bug has already been reported.
+2. **Has it been resolved?** - check if it can be reproduced using the `master` branch or a newer version.
+3. **What is the problem?** - see if you can isolate exactly how to reproduce the issue.
+4. **Include a screenshot or screencast** - you can use [LICEcap](http://www.cockos.com/licecap/) to capture a screencast and save it as an animated gif.
+5. **Tell us all the things** - include as much information as possible (see the template below for a good starting point).
+
+#### Bug Report Template (thanks [Ghost](https://github.com/TryGhost/Ghost) team!)
+```
+Short and descriptive example bug report title
+
+### Issue Summary
+
+A summary of the issue and the browser/OS environment in which it occurs. If
+suitable, include the steps required to reproduce the bug.
+
+### Steps to Reproduce
+
+1. This is the first step
+2. This is the second step
+3. Further steps, etc.
+
+Any other information you want to share that is relevant to the issue being
+reported. Especially, why do you consider this to be a bug? What do you expect to happen instead?
+
+### Technical details:
+
+* Reservations Version: vX.X.X OR master (a1b2c3)
+* Operating System: Mac OS X 10.5.1
```

Comment:
Hmmm... maybe we should find some way to indicate that this is placeholder stuff? Otherwise we could end up with a lot of people claiming to be running OS X 10.5.1 :stuck_out_tongue_closed_eyes: